Providers/nxOMSAutomationWorker/MSFT_nxOMSAutomationWorkerResource.schema.mof

```diff
@@ -1,7 +1,5 @@
-[ClassVersion("1.2.0")]
+[ClassVersion("1.0.0")]
```

Comment:
There were not problems with the local build. I tested it locally by using OMI server push more as mentioned in the one note. Seemed to work without any problems. Can you suggest any more tests that I can run?